project-base/storefront/components/Forms/validationRules.ts

```diff
@@ -170,3 +170,26 @@ export const validateSecondPassword = (t: Translate): BaseSchema => {
         )
         .oneOf([Yup.ref('passwordFirst'), null], t('Passwords must match'));
 };
+
+export const validateNewPassword = (t: Translate): BaseSchema => {
+    return Yup.string()
+        .required(t('Please enter new password'))
+        .min(
+            VALIDATION_CONSTANTS.passwordMinLength,
+            t('Password must be at least {{ count }} characters long', {
+                count: VALIDATION_CONSTANTS.passwordMinLength,
+            }),
+        );
+};
+
+export const validateNewPasswordAgain = (t: Translate): BaseSchema => {
```

Comment:
Just a suggestion. Up to you...

```suggestion
export const validateRepeatedNewPassword = (t: Translate): BaseSchema => {
```